### ISSUE TITLE
Need an easier way to get focus to the main toolbar

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5458]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.x
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
As a screen reader user, or a keyboard-only user, try to navigate and use the main toolbar, i.e. the area highlighted here. You can operate the toolbar with the keyboard if you get focus to it.

![2019-10-02_12-50-53](https://user-images.githubusercontent.com/10569626/66076880-9ba60980-e513-11e9-9aa4-f808daf6754d.png)

### Describe the problem in detail
There is no easy way to get focus to the toolbar.

On server, you can use Ctrl+Alt+F to get focus to the File menu (see Keyboard Shortcuts Help for full list), then use tab several times to maneuver to the toolbar, but that is clunky.

On desktop, there's no real way, other than getting focus somewhere without a keyboard trap and using Shift+Tab to work your way up to the toolbar.

### Describe the behavior you expected
A consistent way to get focus to the toolbar that works on both desktop and server.

**Comment by adamconroy** [comment]
@gtritchie hitting Alt works for me, at least in Windows. Do other OSes not have the same functionality? this is something we could standardize on if there are no conflicts.

**Comment by gtritchie** [comment]
This issue is specifically about having a quick way to get focus to the toolbar, not the main menu.

I mentioned the menus only because on Server you can use tab to get from the menus to the toolbar (because the menus are part of the webpage), but that doesn't work on desktop, where Tab just loops through the menus.

On Mac the standard way to get focus to menus for native apps is Ctrl+F2.

**Comment by adamconroy** [comment]
Oh sorry I mistook the toolbar for the main menu, I'll look into options and standards.

**Comment by gtritchie** [comment]
Not aware of any conventions here, so was planning to create a new command and assign it a keyboard shortcut.

**Comment by ronblum** [comment]
Verified in RStudio Server with Chrome, Firefox, and Safari, and RStudio Desktop. There's no visual indicator that the focus is initially on the New File menu, which you can tell by hitting Enter, Tab, Space, and Right Arrow, so I'll file an issue for that.
